picoquicfirst/picoquicdemo.c

```diff
@@ -293,6 +294,16 @@ int client_loop_cb(picoquic_quic_t* quic, picoquic_packet_loop_cb_enum cb_mode,
             fprintf(stdout, "Waiting for packets.\n");
             break;
         case picoquic_packet_loop_after_receive:
+            if (cb_ctx->cnx_client->is_1rtt_acked) {
```

Comment:
Is probing a new path after the 1RTT packet is acked the right timing?

Reply:
It is probably better to wait until the connection is in state "picoquic_state_ready", which means both client and server are synchronized.